lib/matplotlib/backends/__init__.py

```diff
@@ -14,22 +14,21 @@
            'new_figure_manager', 'backend_version']
 
 backend = matplotlib.get_backend() # validates, to match all_backends
+if backend.startswith('module://'):
+    backend_name = backend[9:]
+else:
+    backend_name = 'matplotlib.backends.backend_' + backend.lower()
 
-def pylab_setup():
-    'return new_figure_manager, draw_if_interactive and show for pylab'
+def get_backend():
```

Comment:
Yes, why not?
On 27 Jun 2015 7:02 pm, "Thomas A Caswell" <notifications@github.com> wrote:

> In lib/matplotlib/backends/__init__.py
> <https://github.com/matplotlib/matplotlib/pull/4143#discussion_r33417288>:
>
> >
> > -def pylab_setup():
> > -    'return new_figure_manager, draw_if_interactive and show for pylab'
> > +def get_backend():
>
> I am still confused. Are you suggesting putting a property on the
> top-level mpl module?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4143/files#r33417288>.
>